src/FormBuilder/resources/js/form-builder/src/App.scss

```diff
@@ -4,3 +4,5 @@
 @import './styles/email-settings';
 @import './styles/components';
 @import './styles/onboarding';
+
+@import './blocks/fields/terms-and-conditions';
```

Comment:
@JoshuaHungDinh you don't need this import since your importing styles directly into the block's edit component.  it _should_ be included in the final stylesheet.

Reply:
@jonwaldstein, hmm just importing into edit.tsx is not applying the styles for me.